pallets/subtensor/src/staking.rs

```diff
@@ -181,10 +174,12 @@ impl<T: Config> Pallet<T> {
         Self::increase_stake_on_coldkey_hotkey_account(&coldkey, &hotkey, stake_to_be_added);
```

Comment:
How come this is `stake_to_be_added` and not `actual_amount_to_stake`? In step 8, it should have returned this value. This is part of the switch to the `fungible` traits from `Currency` -- why has this change been reverted?